texthero/preprocessing.py

```diff
@@ -24,32 +24,39 @@
 
 
 @InputSeries(TextSeries)
-def fillna(s: TextSeries) -> TextSeries:
+def fillna(s: TextSeries, replace_string: Optional[str] = "") -> TextSeries:
     """
-    Replaces not assigned values with empty string.
+    Replaces not assigned values with empty (Default) or given string.
```

Comment:
the default here as well is too much in my opinion. The user will probably recognise, that "" is an empty string

Reply:
Hey, thanks for the comments, this help us learn more. I will commit the changes